build.assets/makefiles/base/network/wait-for-flannel.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+source /etc/container-environment
+
+until [ -e /run/flannel/subnet.env ]; 
+  do echo \"waiting for flannel to start.\"; 
+  sleep 3; 
+done; 
+
+# https://github.com/gravitational/gravity.e/issues/3898
+# The node may not have a default gateway or routes that cover the service subnet
+# This prevents "no route to host" errors when trying to reach the service subnet
+# by creating a dummy interface, and routing the service subnet to this interface
+# we can guarantee that the route exists, and can be NAT to the correct destination
+ip link add flannel.null type dummy
```

Comment:
Is `dummy` module guaranteed to be available or should it be loaded if not?

Reply:
I've never seen it not be loaded and it worked on my systems. To be sure I'll add it to the gravity checkers.